RecoTracker/LSTCore/standalone/README.md

```diff
@@ -1,37 +1,99 @@
+# Source one of the commands below, depending on the site
+source setup.sh # if on UCSD or Cornell
+source setup_hpg.sh # if on Florida
+```
+
+### Setting up LST outside of CMSSW
+
+For this setup, dependencies are still provided from CMSSW through CVMFS (see below for an even more independent setup) but no CMSSW area is setup. This is done by running the following commands.
+
+``` bash
+LST_BRANCH=CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles # Change to the development branch
+git clone --filter=blob:none --no-checkout --depth 1 --sparse --branch ${LST_BRANCH} https://github.com/SegmentLinking/cmssw.git TrackLooper
+cd TrackLooper
+git sparse-checkout add HeterogeneousCore/AlpakaInterface RecoTracker/LSTCore
+git checkout
+cd RecoTracker/LSTCore/standalone/
+# Source one of the commands below, depending on the site
+source setup.sh # if on UCSD or Cornell
+source setup_hpg.sh # if on Florida
+```
+
+#### Running LST in a CVMFS-less setup
+
+The setup scripts included in this repository assume that the [CernVM File System (CVMFS)](https://cernvm.cern.ch/fs/) is installed. This provides a convenient way to fetch the required dependencies, but it is not necessary to run LST in standalone mode. Here, we briefly describe how to build and run it when CVMFS is not available.
+
+The necessary dependencies are CUDA, ROOT, the Boost libraries, Alpaka, and some CMSSW headers. CUDA, ROOT, and Boost, are fairly standard libraries and are available from multiple package managers. For the remaining necessary headers you will need to clone the [Alpaka](https://github.com/alpaka-group/alpaka) and [CMSSW](https://github.com/cms-sw/cmssw) repositories (since the CMSSW repository is extremely large, the instructions just above need to be followed to checkout the strictly required part of it).
+
+Then all that is left to do is set some environment variables. We give an example of how to do this in lnx7188/cgpu-1.
 
-    git clone git@github.com:SegmentLinking/TrackLooper.git
-    cd TrackLooper/
-    # Source one of the commands below, depending on the site
-    source setup.sh # if on UCSD or Cornell
-    source setup_hpg.sh # if on Florida
+```bash
+# These two lines are only needed to set the right version of gcc and nvcc. They are not needed for standard installations.
+export PATH=/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/external/el8_amd64_gcc12/bin:$PATH
+export LD_LIBRARY_PATH=/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/biglib/el8_amd64_gcc12:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/lib/el8_amd64_gcc12:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/external/el8_amd64_gcc12/lib:/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib64:/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib:$LD_LIBRARY_PATH
```

Comment:
I don't quite follow how this can be a part of `when CVMFS is not available`; perhaps the text from below `they can point to local directories` can be added here instead?
- Shouldn't only `external/gcc` and `external/cuda` be necessary instead of the cmssw path?
- are the biglib and cmssw/lib really needed; I'd expect only gcc and cuda are needed

Reply:
Yeah, I think that the extra library paths was because the `gcc` in cvmfs has some preconfigured stuff that depends on the other libraries, but in principle one would just need vanilla `gcc` and `nvcc`.